gtk3/sample/gtk-demo/font_features.rb

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2015 Ruby-GNOME2 Project Team
```

Comment:
Oups,  
it is because I have a script that gets the comments from the c source and create a skeleton with the copyright and launch neovim sessions in order and I have forgottent to update the date.